docs/QSG/tour.md

```diff
@@ -376,7 +378,7 @@ bring-up DAOS servers and clients.
 	0d121c02-a42d-4029-8dce-3919b964b7b3
 
 ### daos container destroy
-
+    $ export export DAOS_CONT=bc4fe707-7470-4b7d-83bf-face75cc98fc
```

Comment:
we should use container label instead of UUID.

Reply:
I'll change everywhere to use cont label.